src/cancelContentPlay.js

```diff
@@ -37,6 +37,16 @@ const cancelContentPlay = function(player) {
       player.el_.style.display = 'block';
       placeholder.remove();
     });
+
+    // On iOS 10 on iPhone, playsinline is added when pinching to return from
+    // fullscreen. Detect fullscreen change, remove placeholder and show player.
+    player.on('fullscreenchange', function () {
+      if (player.el_.hasAttribute('playsinline') && placeholder) {
+        player.el_.style.display = 'block';
```

Comment:
this looks like the same like line 37. We should consider pulling it out into a function. Maybe even have two
* hide player and enable placeholder
* show player and disable placeholder

Doesn't have to change for this PR just something I noticed.